### PR TITLE
Implement tutorial footer window

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
There's an issue with this implementation. Specifically, the 'no background' flag is no longer working, for some reason.

Compare master:
Screenshot_20210324_144617

To this branch:
Screenshot_20210324_144634

The difference is quite prominent.

File: src/OpenLoco/Windows/Tutorial.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    static window_event_list _events;
+
+    static void initEvents();
+
+    // 0x00438CAE
+    window* open()
+    {
+        initEvents();
+
+        auto window = WindowManager::createWindow(
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/Tutorial.cpp:36]: is this not a createWindowCentered?

File: src/OpenLoco/Windows/Tutorial.cpp [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            window->colours[1] = Colour::translucent(skin->colour_07);
+        }
+
+        return window;
+    }
+
+    // 0x00439B3D
+    static void prepareDraw(window* self)
+    {
+        self->widgets[Widx::frame].right = self->width;
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/Tutorial.cpp:59]: width -1

File: src/OpenLoco/Windows/Tutorial.cpp [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+#include "../Graphics/Colour.h"
+#include "../Graphics/Gfx.h"
+#include "../Localisation/FormatArguments.hpp"
+#include "../Objects/InterfaceSkinObject.h"
+#include "../Objects/ObjectManager.h"
+#include "../OpenLoco.h"
+#include "../Ui.h"
+#include "../Ui/WindowManager.h"
+
+#include <array>
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/Tutorial.cpp:11]: not used?

**Review by duncanspumpkin — APPROVED** [review]
see minor comments

File: src/OpenLoco/Windows/Tutorial.cpp (hunk 1 above)
Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/Tutorial.cpp:36]: It's only horizontally centred, at the bottom of the screen. `createWindowCentred` centres on both axes.

**Pull request merged**